spec/menu_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'menu'
+describe Menu do 
+    it 'When initialized menu items and their prices are set' do       
+      expect(subject.dishes).to include({ "Pizza" => 3, "Burger" => 4, "Fries" => 2, "Vodka" => 3.50, "Wine" => 34 })   
+    end
```

Comment:
Could we potentially remove this hard coding, and create a way to supply a "dishes" variable, so that we can use doubles and reduce the length of our lines.